bin/merge

```diff
@@ -83,9 +85,7 @@ if [[ "$merged" == "false" ]]; then
 
     echo "version in ThisBuild := \"${new_version}\"" > version.sbt
 
-    git commit -a --amend --no-edit
-    git checkout $base_ref
-    git merge --no-ff -m "$(printf "Merge branch '%s'\n\nCloses #%s." "$pr_ref" "$pull_request")" $tmp_branch
+    git commit -a --amend --no-edit # amend the merge commit
```

Comment:
Yeah this doesn't work - it amends a real commit (not the merge commit) if there is no merge commit at the `HEAD`.